collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -292,6 +297,14 @@ public void onCreate(Bundle savedInstanceState) {
             showPreviousView();
         });
 
+        if (savedInstanceState == null) {
+            imageLoadingFragment = new ImageLoadingFragment();
+            getFragmentManager().beginTransaction().add(imageLoadingFragment, TAG_IMAGE_LOADING_FRAGMENT).commit();
+        } else {
+            FragmentManager fm = getFragmentManager();
```

Comment:
There is no need to create this object since we use it only once you can directly use `getFragmentManager()` instead.